src/front-end/typescript/lib/pages/opportunity/code-with-us/lib/index.ts

```diff
@@ -116,6 +122,7 @@ export function cwuOpportunityStatusToPastTenseVerb(
     case CWUOpportunityStatus.Draft:
       return "Updated";
   }
+  return "";
```

Comment:
Here as well.

Reply:
taken care of in https://github.com/bcgov/digital_marketplace/pull/430/commits/d6eff27f9821cb4a669aa36c30115a7b995019a9